README.md

```diff
@@ -160,7 +160,9 @@ import Button from '@leafygreen-ui/button';
 | [@lg-tools/update](./tools/update)                                   | [![version](https://img.shields.io/npm/v/@lg-tools/update)](https://www.npmjs.com/package/@lg-tools/update)                                     | ![downloads](https://img.shields.io/npm/dm/@lg-tools/update?color=white)                   | [Docs](http://mongodb.design/component/update/example)                   |
 | [@lg-tools/validate](./tools/validate)                               | [![version](https://img.shields.io/npm/v/@lg-tools/validate)](https://www.npmjs.com/package/@lg-tools/validate)                                 | ![downloads](https://img.shields.io/npm/dm/@lg-tools/validate?color=white)                 | [Docs](http://mongodb.design/component/validate/example)                 |
 
-## Developer Setup
+## Developement
+
+### Setup
 
 1. Node >= 16.20.0 required.
```

Comment:
I feel like this is outdates as well

Reply:
Good catch! It's `>= 18.0.0` now. Will push an update

Reply:
Thank you!